packages/expo/src/Notifications/ExponentNotifications.web.ts

```diff
@@ -110,4 +113,12 @@ export default {
   async getDevicePushTokenAsync(): Promise<{ type: string; data: Object }> {
     return await getDevicePushTokenAsync();
   },
+  
+  async getBadgeNumberAsync(): Promise<number> {
+    return currentBadgeNumber;
+  },
+  async setBadgeNumberAsync(badgeNumber: number): Promise<void> {
+    currentBadgeNumber = badgeNumber
+    badgin.set(badgeNumber)
```

Comment:
Does this clear when set to zero?

Reply:
Yes, it does clear the badge if set to zero. I just released a new version that explicitly checks this case.